packages/msw-addon/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
```

Comment:
We should add `strict: true`. Without it, quite a significant portion of type validation is skipped.

Reply:
Agreed!

Reply:
Fixed!